presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -87,6 +89,7 @@
     private final ResourceGroupId id;
     private final BiConsumer<InternalResourceGroup, Boolean> jmxExportListener;
     private final Executor executor;
+    private final boolean isStaticResourceGroup;
```

Comment:
In my opinion the name is redundant, since the class is called `InternalResourceGroup`.  How about `isStatic`?

Reply:
Sure, I will change.